src/SignalRServiceExtension/TriggerBindings/SignalRParameterAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    /// <summary>
+    /// Mark the parameter as the SignalR parameter that need to bind arguments.
+    /// It's mutually exclusive with <see cref="SignalRTriggerAttribute.ParameterNames"/>. That means
+    /// you can not set <see cref="SignalRTriggerAttribute.ParameterNames"/> and use <see cref="SignalRParameterAttribute"/>
+    /// at the same time.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public class SignalRParameterAttribute : Attribute
+    {
```

Comment:
Also allow specify parameter name

Reply:
I think it's useless. The parameter name is used to match the actual parameter name in method. It should be exactly the name it decorates. We should avoid user using like
```csharp
void Function([SignalRTrigger]InvocationContext context, [SignalRParameter("arg1")]arg0, [SignalRParameter("arg0")]arg1)
```
It's just cause confusing.

Reply:
Like in JSON serialization, you can use JsonProperty to give property a different name (for example a short name). But seems SignalR itself doesn't support this as well. So it should be ok to not support it for now.

Reply:
Then do we really need this? A parameter is by default recognized as a SignalR parameter.

Reply:
Only in class base. It prepared for non-class based function. It's still better than using `ParameterNames` directly